web/cobrands/sass/_base.scss

```diff
@@ -754,7 +754,16 @@ footer {
 
 /*BUTTONS*/
 
-.btn {
+.btn  {
+  @include button-reset;
+}
+
+// Deprecated
+// Try to use `btn btn--primary` etc instead of `btn-primary` where possible.
+.btn-primary,
+.green-btn,
+.btn-danger,
+.red-btn {
   @include button-reset;
```

Comment:
Should this be button-size? Given you then call button-variant on all options of this.